ChangeLog

```diff
@@ -1,3 +1,8 @@
+Version 11.6.1
```

Comment:
Should we hold off creating a new release until the retention/expiry is done?

Reply:
Yeah I was actually thinking the same while raising the quick fix for the retention/expiry

Reply:
I think it's fine to make a bunch of smaller releases here, we just might skip past internally.